features/bundle/index.js

```diff
@@ -13,8 +13,14 @@ function packageFeature({ archiveName, moduleName, artifactPath }, targetDir) {
         moduleName,
         artifactPath
     );
-    const args = ["-r", targetArchive, "."];
-    child_process.execFileSync("zip", args, { cwd: sourceDir });
+    return zip({
+        source: ".",
+        destination: targetArchive,
+        cwd: sourceDir,
+    }).catch((error) => {
+        console.error(error.stack);
```

Comment:
I actually wondered the same :D In short: Yes, it gives a slightly nicer error message, otherwise Node just talks about there being an "uncaught promise failure", this way it's a wee bit nicer, and clear enough where the error is from. The most elegant thing would probably be to turn this into an async function, and catch any exceptions thrown by it below, doing some proper error logging for those cases. But I didn't want to overengineer this little script too much at this time :D